datasources.pkr.hcl

```diff
@@ -0,0 +1,41 @@
+# # Data sources are always treated BEFORE locals and sources.
+data "amazon-ami" "ubuntu-22_04" {
+  access_key      = var.aws_access_key_id
+  secret_key      = var.aws_secret_access_key
```

Comment:
```suggestion
```

as discussed together, we have to use https://developer.hashicorp.com/packer/integrations/hashicorp/amazon#environment-variables